test/index.ts

```diff
@@ -493,7 +493,10 @@ describe('Bigtable', () => {
     });
 
     it('should respect the clusters option', done => {
-      const fakeLocation = 'a/b/c/d';
+      const fakeLocation = Cluster.getLocation_(
```

Comment:
I'm surprised that tests had to be updated, since it looks like this is just a refactor. Does the value of `fakeLocation` actually matter?

It may be better leave the existing tests as is, since it makes it more clear that this is simply a refactor.

Reply:
It looks like we decided just to get rid of the mocks that weren't being used.